<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="RenderFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="generalVolumesTab">
       <attribute name="title">
        <string>Volumes</string>
       </attribute>
       <widget class="QPushButton" name="loadVolume">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>176</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load volume</string>
        </property>
       </widget>
       <widget class="QSlider" name="coronalSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sag</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="volTransparencySlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Transparency</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="axialSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="volColormapBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
       <widget class="QSlider" name="volMaxValSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>230</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QSlider" name="volMinValSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="axialImageCheckBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ax</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="volumesListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>170</width>
          <height>221</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QPushButton" name="unloadVolButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>500</y>
          <width>176</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unload volume</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="volMinValEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="volMaxValEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>240</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="axialEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="coronalEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sagittalEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="volMinValEdit_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QSlider" name="sagittalSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="axialImageCheckBox_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="axialImageCheckBox_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sag</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="generalTractsTab">
       <attribute name="title">
        <string>Tracts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="loadSingleTract">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load tract</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadAllTracts">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAllTracts">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Delete all tracts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="tractsList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tractPropertiesContainer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="generalPropertiesTab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <attribute name="title">
            <string>General</string>
           </attribute>
           <widget class="QCheckBox" name="highlightedOnlyCheckbox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>141</width>
              <height>17</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Highlight selected</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>141</width>
              <height>17</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hide non-selected</string>
            </property>
           </widget>
           <widget class="QPushButton" name="deleteOneTractButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>151</width>
              <height>23</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Delete tract</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>80</y>
              <width>91</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Transparency</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QSlider" name="tractColAlphaSlider">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>110</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Thickness</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QSlider" name="tractThickSlider">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <attribute name="title">
            <string>Tract color light</string>
           </attribute>
           <widget class="QSlider" name="tractColRedSlider">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSlider" name="tractColGreenSlider">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSlider" name="tractColBlueSlider">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>10</y>
              <width>91</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Color by RGB</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>151</width>
              <height>23</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Color by active vol</string>
            </property>
           </widget>
           <widget class="QPushButton" name="tractColDECButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>151</width>
              <height>23</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Color by direction</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
       <zorder>tractPropertiesContainer</zorder>
       <zorder>loadSingleTract</zorder>
       <zorder>loadAllTracts</zorder>
       <zorder>deleteAllTracts</zorder>
       <zorder>tractsList</zorder>
      </widget>
      <widget class="QWidget" name="fodTab">
       <attribute name="title">
        <string>FODs</string>
       </attribute>
       <widget class="QPushButton" name="loadFODButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>176</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load FOD .nii</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteFODButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>176</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Delete FOD</string>
        </property>
       </widget>
       <widget class="QSlider" name="fodSubsampSlider">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>FOD subsampling</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ROIs</string>
       </attribute>
       <widget class="QListWidget" name="ROIsListWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>170</width>
          <height>221</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QPushButton" name="sphereROIButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>176</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add sphere ROI</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
